packages/rest/src/lib/utils/types.ts

```diff
@@ -155,12 +155,23 @@ export interface RateLimitData {
 	 * The HTTP method being performed
 	 */
 	method: string;
+	/**
+	 * The time, in milliseconds, that will need to pass before this specific request can be retried
+	 */
+	retryAfter: number;
 	/**
 	 * The route being hit in this request
 	 */
 	route: string;
 	/**
-	 * The time, in milliseconds, until the request-lock is reset
+	 * The time, in milliseconds, that will need to pass before the sublimit lock for the route resets, and routes that fall under a sublimit
+	 * can be retried
+	 *
+	 * This is only present on certain sublimits, and `0` otherwise
+	 */
+	sublimitTimeout: number;
+	/**
+	 * The time, in milliseconds, until the global route request-lock is reset
```

Comment:
```suggestion
	 * The time, in milliseconds, until this routes request-lock is reset
```
Don't love global here cause it will probably make people think of the 50/s global limit

Reply:
Alternative spelling:
```suggestion
	 * The time in milliseconds until this route's request lock is reset
```